static/js/components/forms/CheckoutForm.js

```diff
@@ -343,7 +344,7 @@ export class InnerCheckoutForm extends React.Component<InnerProps, InnerState> {
                 {coupon ? (
                   <div className="flex-row discount-row">
                     <span>Discount:</span>
-                    <span>{formatPrice(calculateDiscount(item, coupon))}</span>
+                    <span>{formatDiscount(formatNumber(calculateDiscount(item, coupon)))}</span>
```

Comment:
Is `formatNumber` really needed here as `formatDiscount` does that as well?

Reply:
I think you are right, That should be redundant since I've refactored the formatDiscount method itself to take care of this. I'll change that.

Reply:
Addressed in https://github.com/mitodl/mitxpro/pull/2794/commits/68b85d8f36a70bd9238f9534788b05f940d258de